### ISSUE TITLE
Extract multi-search out of pg_search

### ISSUE THREAD

**Issue by kikito** [Casecommons/pg_search#327]
I think that this gem does too much. Have you guys considered splitting `pg_search` into two gems? It seems that the multi-search functionality could be extracted to a separate gem, or even to a wiki page (since it seems to me that multi-searchable can be implemented in terms of `pg_search_scope`)

**Comment by nertzy** [comment]
I agree. I've thought about replacing multisearch with a generator so that there's a way to do more than one multisearch, and so that it would be easier for people to modify the way it works. Naturally I would want to have a deprecation window where both are possible.
